nav2_map_server/test/component/CMakeLists.txt

```diff
@@ -1,54 +1,113 @@
 include_directories(${PROJECT_SOURCE_DIR}/test)
 
 # map_server component test
-ament_add_gtest_executable(test_map_server_node
-  test_map_server_node.cpp
+ament_add_gtest_executable(test_map_server_2d_node
+    test_map_server_2d_node.cpp
   ${PROJECT_SOURCE_DIR}/test/test_constants.cpp
 )
-ament_target_dependencies(test_map_server_node rclcpp nav_msgs)
-target_link_libraries(test_map_server_node
+ament_target_dependencies(test_map_server_2d_node rclcpp nav_msgs nav2_msgs nav2_util)
```

Comment:
Do we really need these two new `nav2_msgs` and `nav2_util` dependencies? test_map_server_2d_node should be unchanged, so I am doubt in there.

Reply:
Yes, those are needed for `server test` as we are not using `I/O` as a dependency, we still, need `get_map` and `load_map` services in there.

Reply:
Agree. These dependencies indeed, required in initial sources too.